websites/N/Netflix/presence.ts

```diff
@@ -178,6 +179,21 @@ presence.on("UpdateData", async () => {
 		prevUrl = document.location.href;
 		browsingTimestamp = Math.floor(Date.now() / 1000);
 	}
+	if (
+		!scriptPushed &&
+		document.readyState !== "complete" &&
+		document.querySelector('[class="loading-children-container"]') &&
+		document.location.pathname.includes("/watch/")
+	) {
+		return;
```

Comment:
So if I am on a /watch/ url and I refresh the page, will the script even be loaded in?